content/sensu-go/5.21/reference/assets.md

```diff
@@ -93,27 +93,69 @@ Follow the steps outlined in [Contributing Assets for Existing Ruby Sensu Plugin
 For further examples of Sensu users who have added the ability to use a community plugin as an asset, see [this Discourse post](https://discourse.sensu.io/t/how-to-use-the-sensu-plugins-kubernetes-plugin/1286).
 {{% /notice %}}
 
-### Asset path
+### Default cache directory
 
-When you download and install an asset, the asset's local path on disk is exposed to asset consumers via environment variables.
-This allows you to retrieve the asset's path from a check, hook, handler, or mutator either as an environment variable or from a check or hook using a custom function for token substitution, `assetPath`.
+system  | sensu-backend                         | sensu-agent
+--------|---------------------------------------|-------------
+Linux   | `/var/cache/sensu/sensu-backend`      | `/var/cache/sensu/sensu-agent`
+Windows | N/A                                   | `C:\ProgramData\sensu\cache\sensu-agent`
 
-For example, if you included a configuration file in the `include` directory of the asset [`sensu-plugins-windows`][4], you can reference the asset from your check in either of these ways:
+If the requested asset is not in the local cache, it is downloaded from the asset URL.
+The Sensu backend acts as an index of asset builds, and does not provide storage or hosting for the build artifacts.
+Sensu expects assets to be retrieved over HTTP or HTTPS.
 
-- `$SENSU_PLUGINS_WINDOWS/include/config.yaml`
-- `${{assetPath "sensu-plugins-windows"}}/include/config.yaml`
+### Example asset structure
+
+{{< code shell >}}
+sensu-example-handler_1.0.0_linux_amd64
+├── CHANGELOG.md
+├── LICENSE
+├── README.md
+└── bin
+  └── my-check.sh
+└── lib
+└── include
+{{< /code >}}
+
+## Asset path
+
+When you download and install an asset, the asset files are saved to a local path on disk.
+Most of the time, you won't need to know this path &mdash; except when you want to provide an argument to a command that uses the asset.
+In that case, you *will* need to know the full explicit path to the asset files.
+
+The asset path includes the asset's checksum, which changes every time the asset is updated.
+This would normally require you to manually update the commands for any of your checks, handlers, hooks, or mutators that consume the asset.
+However, the local path on disk is exposed to asset consumers via [environment variables][14] and the [`assetPath` custom function for token substitution][17], which allows you to avoid manual updates.
```

Comment:
```suggestion
However, because the asset directory path is exposed to asset consumers via [environment variables][14] and the [`assetPath` custom function for token substitution][17], you can avoid these manual updates.
```